### DISCUSSION TITLE
how to start chevereto in macos

### DISCUSSION THREAD

**Discussion by aylwynlake** [chevereto/chevereto#64] [Q&A] [answered]
I want to deploy chevereto on my mac. How to start chevereto in macos? Is there any tutorials

**Comment by rodber** [comment, marked as answer]
Chevereto requires MySQL, PHP and a web server. You can provide that directly installing each one of these requirements. You can also use a bundle (like mamp).

I recommend using Docker, that includes pretty much everything but you will require to learn Docker.
